README.md

```diff
@@ -293,7 +293,7 @@ Estimate of the total amount of disk space required to install the named package
 
 #### options.depends, recommends, suggests, enhances, preDepends
 Type: `Array[String]`
-Default: see [`src/installer.js`](https://github.com/unindented/electron-installer-debian/blob/master/src/installer.js#L181)
+Default: see `src/installer.js` for [`depends`](https://github.com/unindented/electron-installer-debian/blob/16296e7/src/installer.js#L115-L148), [`recommends`, `suggests`, `enhances`, and `preDepends`](https://github.com/unindented/electron-installer-debian/blob/16296e7/src/installer.js#L181-L192)
```

Comment:
I'm ok with this. I originally merged the `depends` and `trashDepends` functions so we didn't have two links here, but I guess having multiple links is fine and might be better this way